.github/workflows/CI.yml

```diff
@@ -1,7 +1,9 @@
 name: CI
 on:
-  - push
-  - pull_request
+  push:
+    branches: [main, master]
+    tags: ["*"]
+  pull_request:
```

Comment:
Taken from https://github.com/JuliaLang/Example.jl/blob/master/.github/workflows/ci.yml. I think this is better since it makes CI half as expensive. :)